### PR TITLE
Update supported Python & Django versions

### PR DESCRIPTION
**Pull request by ulgens**
Python 3.7 and Django 2.2. are EOL.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,14 +36,15 @@ def _get_long_description():
...
-            'Programming Language :: Python :: 3.7',
             'Programming Language :: Python :: 3.8',
             'Programming Language :: Python :: 3.9',
             'Programming Language :: Python :: 3.10',
+            'Programming Language :: Python :: 3.11',
+            'Programming Language :: Python :: 3.12',
         ],
         install_requires=[
-            'Django>=2.2',
+            'Django>=3.2',
```
Comment (hartwork) [inline setup.py:46]: Did you intend to say 4.2 here? Why 3.2?

https://endoflife.date/django

Comment (hartwork) [reply setup.py:46]: @ulgens any news?

Comment (ulgens) [reply setup.py]: Updated now.

**Review by hartwork — APPROVED** [review]
@ulgens thank you! :+1:

**Pull request merged**